src/widgets/emby/component.jsx

```diff
@@ -215,6 +219,7 @@ export default function Component({ service }) {
 
   const enableBlocks = service.widget?.enableBlocks;
   const enableNowPlaying = service.widget?.enableNowPlaying ?? true;
+  const enableUser = service.widget?.enableUser ?? true;
```

Comment:
```suggestion
  const enableUser = !!service.widget?.enableUser;
```

This also currently makes it default to true